### DISCUSSION TITLE
Additional Organization Tricks?

### DISCUSSION THREAD

**Discussion by chase1124** [cxw42/TabFern#328] [General]
Hi,

First, tabfern has changed my life as a "tab collector" and it has changed my life over years of use. Thank you!

One problem is that I have so many folders that it's very hard to resurface things from the past (I group browser windows / tabfern folders by research projects, for example). I guess the simplest useful addition would either be a folder that applies only to tabfern for higher level organization of topics or else some kind of search feature so you could name tabfern folders by some pattern and resurface them as groups. BUT I also understand development for this project is probably over :) so I guess I'm wondering if there are any tricks other people are using or sadly if people have moved on to some other plugin? Thinking about it now I guess I could manually sort the folders and use the sort although that's terrifying since the generally date-ordered sort often proves helpful

**Comment by cxw42** [comment]
Development is not over, but I have much less time than I used to :( .  If you are a JavaScript developer or know any JS devs, contributions are welcome!  Hierarchical organization, for example, is open as #34.

You can use Ctl+F in the TabFern window to search for the text of window notes and titles.  You could put search keywords in the notes if you wanted to, for example.
